packages/client/src/generation/TSClient/Input.ts

```diff
@@ -178,7 +183,7 @@ export type ${this.getTypeName()} = ${wrapWithAtLeast(body, type)}`
     if (this.genericsInfo.inputTypeNeedsGenericModelArg(this.type)) {
       return `${this.type.name}<$PrismaModel = never>`
     }
-    return this.type.name
+    return `${this.type.name}<$PrismaModel = never>`
```

Comment:
I think now `$PrismaModel=never` would be added in all cases, regardless of the check above? Is it intended?